ci/provisioning/post_provision_config_nodes.sh

```diff
@@ -73,6 +78,41 @@ dump_repos() {
         done
 }
 
+send_mail() {
+    local subject="$1"
+    local message="$2"
+    echo $message 2>&1 | mail -s $subject -r "$HOSTNAME"@intel.com "$OPERATIONS_EMAIL"
+}
+
+retry_cmd() {
+    local attempt=0
+    local rc=0
+    while [ $attempt -lt $DAOS_STACK_RETRY_COUNT ]; do
+        if time "$@"; then
```

Comment:
(lint) Couldn't find 'fi' for this 'if'. [SC1046]
(lint) Couldn't parse this if expression. Fix to allow more checks. [SC1073]